recipes/gm2calc/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.30.0"
+
+
+class Gm2calcConan(ConanFile):
+    name = "gm2calc"
+    license = "GPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/GM2Calc/GM2Calc"
+    description = "C++ library to calculate the anomalous magnetic moment of the muon in the MSSM"
+    topics = ("HEP")
```

Comment:
Maybe add more topics?

Reply:
Good point! Is there a list of common topics, or can I just invent some that I think are suitable?

Reply:
Usually:
- conan
- <lib name>
- any keyword related to your library (hep, high-energy, magnetic-moment, things like that)

Reply:
Thanks!